src/content/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model.mdx

```diff
@@ -13,33 +13,53 @@ watermark: |-
 
 This tutorial will walk you through some common procedures for managing users on the New Relic One user model.
 
+
 ## Requirements [#requirements]
 
 Some notes on who this tutorial is for and requirements:
 
-* This tutorial is for users on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models).
+* For users on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models).
 * Creating access grants requires Pro or Enterprise edition. For details, see [user management requirements](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#requirements).
 * [Role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#standard-roles) requirements:
   * Using the **Organization and access** UI to manage group access requires **Authentication domain manager** and **Organization manager** roles.
   * Using the **User management** UI to add/edit users requires **Authentication domain manager** role.
 
 ## Overview [#overview]
 
-Before using this tutorial, we recommend you read:
+Before using this tutorial, we recommend you check out:
 
-* [Where to find the user management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where)
 * [Understand what an access grant is](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)
+* [Some videos of using the user management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/user-mgmt-videos)
 
 This tutorial will walk you through how to:
 
+* [Add users](#add-users)
 * [View available accounts](#accounts)
 * [How to grant groups access to roles and accounts](#group-access) 
 * [How to create custom roles](#roles)
 * [How to add users to groups](#add-users)
 
+## Add users [#add-users]
+
+Adding users to New Relic is a separate process from creating access grants. In other words, you can do either procedure first. The key point is that if you're trying to give a user access to a role or an account that they don't yet have, they won't have that access until they're both a) in a group and b) that group has an access grant that gives them that role and account access. 
```

Comment:
"creating access grants" feels unnecessarily complicated/tortured. Can't we just say "granting access"?